README.md

```diff
@@ -1,20 +1,7 @@
 # Cardillo
 [![License](https://img.shields.io/badge/License-BSD_3--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause) [![License](https://img.shields.io/badge/code%20standard-PEP8-black)](https://www.python.org/dev/peps/pep-0008/)
 
-
-## Table of Contents
-
-- [Cardillo](#cardillo)
-  - [Table of Contents](#table-of-contents)
-  - [Requirements](#requirements)
-  - [Installation](#installation)
-  - [Virtual environments](#virtual-environments)
-  - [Documentation](#documentation)
-  - [Developing](#developing)
-    - [Pre-requisites](#pre-requisites)
-    - [Naming Conventions](#naming-conventions)
-    - [Useful links](#useful-links)
-    - [Further information](#further-information)
+Cardillo is an open-source simulation framework for flexible multi-body systems with frictional contacts and impacts. The software is developed by teams from the University of Stuttgart, the Eindhoven University of Technology (TU/e) and the University of Erlangen-Nuremberg (FAU).
```

Comment:
Friedrich-Alexander-Universität Erlangen-Nürnberg (FAU), sorry, but corporate identity does not allow translations or shortening...